AmyShackles.md

```diff
@@ -1 +1,3 @@
 Amy Victoria Shackles
+
+I really like writing programs in C.  I'm currently trying to build a Lisp interpreter.  So far, it's been mostly reviewing C, but forward progress will happen soon, I'm sure!
```

Comment:
That's awesome! I have never written any C, but I would love to take a crack at it some day.

Reply:
It's definitely not a language for everyone.  I love it because it's strongly typed and more structured.  http://buildyourownlisp.com/contents is the thing I'm currently working through - would recommend.  Does a good job of covering the basics of C without getting into the overwhelming parts of C early on.  ;)

Reply:
I will definitely take a look! I am getting more and more into strongly typed languages. I have done a few little exercises with ReasonML, and I love it.